src/linear_algebra/affine_space/finite_dimensional.lean

```diff
@@ -24,13 +24,16 @@ open_locale big_operators classical affine
 
 section affine_space'
 
-variables (k : Type*) {V : Type*} {P : Type*} [field k] [add_comm_group V] [module k V]
+variables (k : Type*) {V : Type*} {P : Type*} [add_comm_group V]
           [affine_space V P]
 variables {ι : Type*}
 include V
 
 open affine_subspace finite_dimensional module
 
+section
+variables [division_ring k] [module k V]
```

Comment:
I'd be tempted to move the typeclasses above too, making this
```suggestion
variables [division_ring k] [add_comm_group V] [module k V] [affine_space V P]
```
Just so that the argument order matches what we usually use

Reply:
Good call.